container-host-files/etc/hcf/config/role-manifest.yml

```diff
@@ -1350,6 +1374,7 @@ roles:
     scaling:
       min: 1
       max: 254
+      ha: 3
```

Comment:
Why do we need 3 cells for diego-cell? I don't think anything here requires consensus.

Reply:
Oh you mentioned this. Put a comment in the yaml?

Reply:
Done in https://github.com/SUSE/scf/pull/1159/commits/ac8eb64e69a53282a8384d1082c2106061e9f845

Reply:
Added comment in https://github.com/SUSE/scf/pull/1159/commits/ac8eb64e69a53282a8384d1082c2106061e9f845